query-coordinator/src/test/scala/com/socrata/querycoordinator/TestQueryRewriter.scala

```diff
@@ -407,4 +408,15 @@ class TestQueryRewriter extends TestQueryRewriterBase {
 
     rewrites should have size 1
   }
+
+  test("rewrite where with where removed") {
```

Comment:
I know this is still in progress, but just to toss some brainstorming out... it would be good to see a test showing it doesn't rewrite when the query doesn't have a where or has a non-matching where, and either rewrites or doesn't as appropriate if there is a compound where (not sure where the code landed on supporting that).

Reply:
That could be covered by existing tests.  But I also added more negative tests since then.  I am going to stop adding more tests now.  Let me know if there are more specifics that you want me to cover.

Reply:
Those look great, thanks.  I find it very useful to have the examples explicit.